modules/iam-role-for-service-accounts-eks/policies.tf

```diff
@@ -583,6 +584,19 @@ data "aws_iam_policy_document" "karpenter_controller" {
     actions   = ["iam:PassRole"]
     resources = var.karpenter_controller_node_iam_role_arns
   }
+
+  statement {
+    sid       = "KarpenterEventPolicySQS"
+    effect    = "Allow"
+    resources = ["arn:aws:sqs:${data.aws_region.current.name}:${local.account_id}:${var.karpenter_controller_cluster_id}"]
```

Comment:
lets add a variable for this `karpenter_sqs_queue_arn` since we are not guaranteed that the queue name will always bee the cluster name